build.sbt

```diff
@@ -0,0 +1,17 @@
+name := "WebServiceClients"
+
+version := "1.0"
+
+lazy val `webserviceclients` = (project in file(".")).enablePlugins(PlayScala)
+
+scalaVersion := "2.11.7"
+
+libraryDependencies ++= Seq( jdbc , cache , ws   , specs2 % Test,
+  "org.webjars.npm" % "jquery" % "2.1.4",
```

Comment:
Why the npm webjar... that seems like a mistake, but good for now. Will need to look at it later.